src/docs/MLOS/MLOSUserGuide.adoc

```diff
@@ -331,24 +331,493 @@ The following workflows represent some mathematical functions that can be optimi
 
 *Himmelblau_Function:* is a multi-modal function, used to test the performance of optimization algorithms.
 
-//.Himmelblau_Function_Workflow variables
-//[cols="2,5,2"]
-//|===
-//| *Variable name* | *Description* | *Type*
-//| `DOCKER_ENABLED`
-//|If True,  the workflow will be executed on a Docker container. link:https://hub.docker.com/r/activeeon/dlm3/[activeeon/dlm3] image is used by default. If False, the required libraries should be installed in the different nodes that will be used.
-//| Boolean (default=True)
-//| `DOCKER_IMAGE`
-//| Specifies the docker image that will be used to execute the task.
-//| String (default="activeeon/dlm3")
-//| `INPUT_VARIABLES`
-//| Specifies the x and y values.
-//| JSON format (default={"x": 3.5976705887960527, "y": -1.8471754188451843})
-//| `TOKEN`
-//|  Specifies the id for the tested parameters.
-//| String (default=-1)
-//|===
```

Comment:
Why this block was `commented` and then `removed` ?

Reply:
It is commented as for the other objective examples workflows, we didn't provide a detailed description for the variables. So, we have done the same for Himmelblau to keep the documentation lite and homogeneous.